tendrl/commons/jobs/__init__.py

```diff
@@ -209,7 +198,7 @@ def process_job(jid):
                     obj_name, flow_name)
             else:
                 runnable_flow = current_ns.ns.get_flow(flow_name)
-
+            time.sleep(3)
             job = job.load()
```

Comment:
this will avoid same job picked by different nodes

Reply:
So this sleep for avoiding clash of threads. Fine.

Reply:
> So this sleep for avoiding clash of threads. Fine.

yes, even all nodes are pick up the job at the same time, this sleep will stop the node execution for 3 sec and tell them who has locked the job recently. So only the node which is locked the job can continue the execution.